src/DummyTest.java

```diff
@@ -1,9 +1,101 @@
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.BeforeEach;
+
+import org.junit.jupiter.params.provider.NullSource;
+
+
 
 import static org.junit.jupiter.api.Assertions.*;
 
 // FIXME: remove this class
 public class DummyTest {
+    static Entity entity1;
+    static Entity entity2;
```

Comment:
Ezek nincsenek használva + static nem kell ehhez + láthatósági jelző hiányzik.

Láthatóság legyen ott a metódusoknál viszont.